lib/hyrax/configuration.rb

```diff
@@ -319,6 +319,12 @@ def active_deposit_agreement_acceptance
       @active_deposit_agreement_acceptance
     end
 
+    attr_writer :show_deposit_agreement
+    def show_deposit_agreement
```

Comment:
There are a bunch of existing boolean options `enable_ffmpeg`, `display_microdata`, etc., and none of them have question marks now.  Should all of them have question marks?